cmd/collector/app/builder/builder_flags.go

```diff
@@ -71,11 +83,15 @@ type CollectorOptions struct {
 func AddFlags(flags *flag.FlagSet) {
 	flags.Int(collectorQueueSize, app.DefaultQueueSize, "The queue size of the collector")
 	flags.Int(collectorNumWorkers, app.DefaultNumWorkers, "The number of workers pulling items from the queue")
-	flags.Int(collectorPort, ports.CollectorTChannel, "The TChannel port for the collector service")
-	flags.Int(collectorHTTPPort, ports.CollectorHTTP, "The HTTP port for the collector service")
-	flags.Int(collectorGRPCPort, ports.CollectorGRPC, "The gRPC port for the collector service")
-	flags.Int(collectorZipkinHTTPort, 0, "The HTTP port for the Zipkin collector service e.g. 9411")
-	flags.Bool(collectorGRPCTLS, false, "Enable TLS for the gRPC collector port")
+	flags.Int(collectorPort, ports.CollectorTChannel, "The TChannel port for the collector service - (deprecated) see - "+collectorTChanHostPort)
+	flags.Int(collectorHTTPPort, ports.CollectorHTTP, "The HTTP port for the collector service - (deprecated) see -"+collectorHTTPHostPort)
+	flags.Int(collectorGRPCPort, ports.CollectorGRPC, "The gRPC port for the collector service - (deprecated) see -"+collectorGRPCHostPort)
```

Comment:
we usually place "annotations" like `(experimental)` at the front of the string, should we do the same with `(deprecated)`?

Reply:
Also, I think if we do that, then the option does not require explanation, simply 

```
(deprecated) please use collector.grpc-server.host-port
```

Reply:
IIRC we already do it with deprecated.